From d469b385f7c91b41e51a7a274926e14e9c2963be Mon Sep 17 00:00:00 2001
From: Petr Gajdos <pgajdos@suse.cz>
Date: Mon, 18 Feb 2013 17:16:05 +0100
Subject: Fix the build with libpng16

Include string.h explicitly in gxps-png-writer.c since png.h no longer
includes it.

diff --git a/tools/gxps-png-writer.c b/tools/gxps-png-writer.c
index 369e0e0..263b8ae 100644
--- a/tools/gxps-png-writer.c
+++ b/tools/gxps-png-writer.c
@@ -22,6 +22,7 @@
 #include "gxps-png-writer.h"
 #include <png.h>
 #include <stdint.h>
+#include <string.h>
 
 /* starting with libpng15, png.h no longer #includes zlib.h */
 #ifndef Z_BEST_COMPRESSION
-- 
cgit v0.10.1

